Controllers/Backend/Import.php

```diff
@@ -237,7 +237,29 @@ public function assignRemoteToLocalCategoryAction()
         }
 
         try {
-            $this->getImportService()->importRemoteCategory($localCategoryId, $remoteCategoryKey, $remoteCategoryLabel);
+            $categories = $this->getImportService()->createCategoriesFromRemoteCategoires($localCategoryId, $remoteCategoryKey, $remoteCategoryLabel);
```

Comment:
In the past ```$this->getImportService()->importRemoteCategory($localCategoryId, $remoteCategoryKey, $remoteCategoryLabel);``` was called to create necessary entities and store them into DB.

Now ```createCategoriesFromRemoteCategoires``` will generate categoryTree and extract category keys  ```return $this->autoCategoryResolver->convertTreeToKeys($remoteCategoryNodes, $localCategory);```. Then this action returns response success true, but categories are not imported.